mutiny-core/src/labels.rs

```diff
@@ -155,6 +155,20 @@ pub trait LabelStorage {
         }
         Ok(None)
     }
+    /// Finds a contact that has the given npub
+    fn get_contact_for_npub(
+        &self,
+        npub: XOnlyPublicKey,
+    ) -> Result<Option<(String, Contact)>, MutinyError> {
+        // todo this is not efficient, we should have a map of npub to contact
```

Comment:
Does this todo need to happen here?

Reply:
It's okay for now, not terribly inefficient, but we should fix before going to prod with this